sdk/test/metrics/async_metric_storage_test.cc

```diff
@@ -78,8 +78,8 @@ TEST_P(WritableMetricStorageTestFixture, TestAggregation)
                      opentelemetry::common::SystemTimestamp(std::chrono::system_clock::now()));
 
   storage.Collect(collector.get(), collectors, sdk_start_ts, collection_ts,
-                  [&](const MetricData data) {
-                    for (auto data_attr : data.point_data_attr_)
+                  [&](const MetricData metric_data) {
+                    for (auto data_attr : metric_data.point_data_attr_)
```

Comment:
Not a problem of this PR, should it be `auto& data_attr` here to avoid copy?

Reply:
Done